index.js

```diff
@@ -67,7 +67,7 @@ module.exports = options => {
 				insert(result, 'alias', alias, key);
 			});
 
-			if (props.default) {
+			if (Object.prototype.hasOwnProperty.call(props, 'default')) {
```

Comment:
XO threw a `no-prototype-builtins` error when calling `hasOwnProperty`. That can be overridden, but I opted not to since there's no XO configuration in the project yet.